packages/builder/src/components/common/Input.svelte

```diff
@@ -5,13 +5,24 @@
   export let textAlign = "left"
   export let width = "160px"
   export let placeholder = ""
+  export let suffix = ""
+  export let onChange = val => {}
   
   let centerPlaceholder = textAlign === "center"
 
   let style = buildStyle({ width, textAlign })
+
+  function handleChange(val) {
+    value = val
+    let _value = suffix ? value + suffix : value
+    onChange(_value)
```

Comment:
`suffix` defaults to an empty string, so you could probably get rid of 17 and just do

```js
onChange(value + suffix)
```

Reply:
Nice tip 👌 well spotted